compute/cloud-client/src/main/java/compute/ListAllInstances.java

```diff
@@ -39,9 +40,21 @@ public static void listAllInstances(String project) throws IOException {
     // the `instancesClient.close()` method on the client to 
     // safely clean up any remaining background resources.
     try (InstancesClient instancesClient = InstancesClient.create()) {
-      // Listing all instances for the project
```

Comment:
to confirm, are we replacing this example? Or should there be two?

Reply:
No, we are not replacing it. I felt that the comment was redundant and hence removed it.